msteams-platform/m365-apps/overview.md

```diff
@@ -23,7 +23,12 @@ The Teams app platform continues to evolve and expand holistically into the Micr
 | [**Link unfurling (including Stage View)**](../tabs/tabs-link-unfurling.md) | `composeExtensions.messageHandlers` | Web, Desktop | Web, Desktop | - | See notes on [link unfurling](extend-m365-teams-message-extension.md#link-unfurling) and [Stage View](extend-m365-teams-message-extension.md#stage-view)|
 | [**Office Add-ins**](/office/dev/add-ins/develop/json-manifest-overview) (preview) | `extensions` | - | Web, Desktop | - | Only available in [devPreview](../resources/schema/manifest-schema-dev-preview.md) manifest version. See [notes](#office-add-ins-preview).|
 
-Enrollment to [Microsoft 365 Targeted Release](/microsoft-365/admin/manage/release-options-in-office-365) and [Microsoft 365 Apps update channel](/deployoffice/change-update-channels) requires admin opt-in for the entire organization or selected users. For more information, see [Manage access to Teams apps across Microsoft 365](/microsoftteams/manage-third-party-teams-apps). Update channels are device specific and apply only to installations of Microsoft 365 running on Windows.
+Enrollment to [Microsoft 365 Targeted Release](/microsoft-365/admin/manage/release-options-in-office-365) and [Microsoft 365 Apps update channel](/deployoffice/change-update-channels) requires admin opt-in for the entire organization or selected users.
+
+> [!NOTE]
+> For additional admin options and guidance on your extended Teams app, please see the documentation [*'Teams apps that work on Outlook and Microsoft 365'*](/microsoft-365/admin/manage/teams-apps-work-on-outlook-and-m365)
+
+Update channels are device specific and apply only to installations of Microsoft 365 running on Windows.
```

Comment:
This sentence should be with the sentences above on TR and M365 Apps update channels before the !NOTE